MLOps.NET.Azure/Entities/Experiment.cs

```diff
@@ -17,6 +18,6 @@ public Experiment(string experimentName)
 
         public string ExperimentName { get; set; }
 
-        public List<Run> Runs { get; set; }
+        public IList<IRun> Runs { get; set; }
```

Comment:
Sorry about that, the comment appears to have disappeared. The comment was about this property. It should probably actually be removed as we don't use it in the NoSQL scenario. That leaves the question though how we do with other implementations that need it. I see it that we can either:

1. Keep our entities flat
2. Keep this property but populate it in the NoSQL case after fetching an experiment

Reply:
We might segregate the interface, therefore we implement only those that are needed?

Just to double-check (because I'm missing the big picture here), is that making sense if I split IExperiment into IExperiment (without Run) and IRunableExperiment (having Run)?

Reply:
I think they all in theory have runs, just not as part of their composition :)
To keep it simple now, could we make them flat? E.g. not having a list of Runs?

Reply:
but only one? like public IRun Run { get; set;} ?

Reply:
All runs would have an associated ExperimentId, so we can fetch them using that instead. I think we probably also want to add ExperimentId as a field to IRun, and RunId to IMetric

Reply:
Happy to open a PR to this PR with the changes I'm suggesting if that helps to clarify it

Reply:
Oh, first take a look at my latest commit.

Reply:
Love it, this is exactly what I was thinking we could start with!

Reply:
One last thing @dcostea and then I think we're good, could you set the ExperimentId in the Run class' constructor?

Reply:
done!